index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>CanJS GitHub Issues To-Do List</title>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+  <link rel="stylesheet" href="app.css">
+</head>
+<body>
+
+<script type="text/stache" id="app-template">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-8 col-md-offset-2">
+        <github-issues {page-title}="pageTitle" />
+      </div>
+    </div>
+  </div>
+</script>
+
+<script type="text/stache" id="github-issues-template">
+  <h1 class="page-header text-center">
+    {{pageTitle}}
+  </h1>
+
+  <form ($submit)="send(%event, issuesPromise.value)">
+    <div class="form-group">
+      <label for="title" class="sr-only">Issue title</label>
+      <input class="form-control" id="title" placeholder="Issue title" type="text" {($value)}="title" />
+    </div>
+    <div class="form-group">
+      <label for="body" class="sr-only">Issue description</label>
+      <textarea class="form-control" id="body" placeholder="Issue description" {($value)}="body"></textarea>
+    </div>
+    <button class="btn btn-primary" type="submit">Submit issue</button>
+  </form>
+
+  <div class="list-group">
+    {{#if issuesPromise.isPending}}
+      <div class="list-group-item list-group-item-info">
+        <h4>Loading…</h4>
+      </div>
+    {{/if}}
+    {{#if issuesPromise.isRejected}}
+      <div class="list-group-item list-group-item-danger">
+        <h4>Error</h4>
+        <p>{{issuesPromise.reason}}</p>
+      </div>
+    {{/if}}
+    {{#if issuesPromise.isResolved}}
+      {{#if issuesPromise.value.length}}
```

Comment:
you can use `{{#each .value}} LOOP {{else}} NO ITEMS{{/each}}` and save yourself some nesting.